src/devices/Lcm1602a1/Flags.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.CharacterLCD
+{
+    // Command flags are found on page 24/25 of the HD4478U spec.
+
+    [Flags]
+    internal enum DisplayEntryMode : byte
```

Comment:
Even though having the flags like the way you are setting them makes more sense, I kind if like the fact of having the names match exactly what the datasheet uses for the names. If you only did it for readability, I think it would be better to match the datasheet in case some consumer wanted to do something advanced based on the data sheet.

Reply:
I have the comments about what they match up to. The datasheet flag names are things like "S". There are no definitions for groups of flags. I think it makes reading the code much easier to use human readable names- you'll note that the original names were made up constants from some very old source that everything seems to have derived from (i.e. they don't match the datasheet).  If you have readable names people can understand what is going on without having to dig out the datasheet or totally rely on comments.